core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -16,19 +16,269 @@
  */
 package kafka.admin
 
-import org.junit.Assert._
-import org.junit.Test
-import kafka.utils.Logging
-import kafka.utils.TestUtils
-import kafka.zk.{ConfigEntityChangeNotificationZNode, DeleteTopicsTopicZNode, ZooKeeperTestHarness}
+import kafka.admin.TopicCommand.{TopicCommandOptions, ZookeeperTopicService}
 import kafka.server.ConfigType
-import kafka.admin.TopicCommand.TopicCommandOptions
-import org.apache.kafka.common.errors.TopicExistsException
+import kafka.utils.ZkUtils.getDeleteTopicPath
```

Comment:
ZkUtils is deprecated. We need to use kafka.zk.DeleteTopicsTopicZNode

Reply:
Replaced it, thanks.